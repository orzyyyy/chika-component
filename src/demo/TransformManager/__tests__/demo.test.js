import React from 'react';
import { mount } from 'enzyme';
import TransformManagerDemo from '..';

describe('TransformManagerDemo', () => {
  it('render correctly', () => {
    const wrapper = mount(<TransformManagerDemo />);
    expect(wrapper).toMatchSnapshot();
  });
});
