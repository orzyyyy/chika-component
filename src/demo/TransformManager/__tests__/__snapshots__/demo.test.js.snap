// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TransformManagerDemo render correctly 1`] = `
<TransformManagerDemo>
  <div
    className="TransformManagerDemo"
  >
    <Button
      activeStyle={Object {}}
      disabled={false}
      inline={false}
      loading={false}
      onClick={[Function]}
      prefixCls="am-button"
      size="large"
      type="primary"
    >
      <TouchFeedback
        activeClassName="am-button-active"
        activeStyle={Object {}}
        disabled={false}
      >
        <a
          aria-disabled={false}
          className="am-button am-button-primary"
          onClick={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchCancel={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
        >
          <span
            key=".0"
          >
            next order
          </span>
        </a>
      </TouchFeedback>
    </Button>
    <Button
      activeStyle={Object {}}
      disabled={false}
      inline={false}
      loading={false}
      onClick={[Function]}
      prefixCls="am-button"
      size="large"
    >
      <TouchFeedback
        activeClassName="am-button-active"
        activeStyle={Object {}}
        disabled={false}
      >
        <a
          aria-disabled={false}
          className="am-button"
          onClick={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchCancel={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
        >
          <span
            key=".0"
          >
            reset
          </span>
        </a>
      </TouchFeedback>
    </Button>
    <Button
      activeStyle={Object {}}
      disabled={false}
      inline={false}
      loading={false}
      onClick={[Function]}
      prefixCls="am-button"
      size="large"
    >
      <TouchFeedback
        activeClassName="am-button-active"
        activeStyle={Object {}}
        disabled={false}
      >
        <a
          aria-disabled={false}
          className="am-button"
          onClick={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchCancel={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
        >
          <span
            key=".0"
          >
            group2-0
          </span>
        </a>
      </TouchFeedback>
    </Button>
    <TransformManager
      currentGroup="group1"
      currentOrder={0}
    >
      <div
        className="TransformManager"
      >
        <div
          className="Transform-item"
          group="group1"
          key="group1-0"
          style={
            Object {
              "transform": "translate3d(0%, 0, 0)",
            }
          }
        >
          group1-0
        </div>
        <div
          className="Transform-item"
          group="group1"
          key="group1-1"
          style={
            Object {
              "transform": "translate3d(100%, 0, 0)",
            }
          }
        >
          group1-1
        </div>
        <div
          className="Transform-item"
          group="group1"
          key="group1-2"
          style={
            Object {
              "transform": "translate3d(200%, 0, 0)",
            }
          }
        >
          group1-2
        </div>
      </div>
    </TransformManager>
  </div>
</TransformManagerDemo>
`;
